oak-it/src/test/java/org/apache/jackrabbit/oak/composite/CompositeNodeStoreTest.java

```diff
@@ -18,9 +18,9 @@
  */
 package org.apache.jackrabbit.oak.composite;
 
-import static com.google.common.base.Predicates.compose;
-import static com.google.common.collect.Iterables.filter;
 import static com.google.common.collect.Lists.newArrayList;
```

Comment:
imho that static import of com.google.common.collect.Lists.newArrayList should also be replaced, no?

Reply:
I tried, but once we do that consistenly, the diffs get really big. Let's take care of these things later on.